app/controllers/state_file/questions/az_charitable_contributions_controller.rb

```diff
@@ -0,0 +1,11 @@
+module StateFile
+  module Questions
+    class AzCharitableContributionsController < QuestionsController
+      private
+
+      def form_params
+        params.require(:state_file_az_charitable_contributions_form).permit([:charitable_contributions, :charitable_cash, :charitable_noncash])
+      end
```

Comment:
I would have thought that these params were permitted automatically by something in the QuestionsController. I believe you can delete this method, unless you tried that before and it failed. If it failed, I'm curious to know why.